### PR TITLE
Fix position and responsability of logs into horusec

### PR DESCRIPTION
**Pull request by wiliansilvazup**
Some error messages were with the warning type naming, so I changed them so they all had the correct naming.
Another change was that the error message file had two types of messages:
    * Return messages in functions like fmt.Errorf or errros.New
    * Error log messages in functions such as logger.LogError or logger.LogErrorWithLevel
Signed-off-by: wilian <wilian.silva@zup.com.br>

### CODE REVIEW

File: internal/usecases/cli/cli.go [hunk 1]
```diff
@@ -125,10 +125,10 @@ func (au *UseCases) validateTextOutputFilePath(cfg *config.Config) error {
 
 func (au *UseCases) validateFilePathAndExtension(cfg *config.Config, extension string) error {
 	if filepath.Ext(cfg.JSONOutputFilePath) != extension {
-		return fmt.Errorf("%snot valid file of type %s", messages.MsgErrorJSONOutputFilePathNotValid, extension)
+		return fmt.Errorf("%s %s", messages.MsgErrorJSONOutputFilePathNotValidExtension, extension)
 	}
 	if output, err := filepath.Abs(cfg.JSONOutputFilePath); err != nil || output == "" {
-		return errors.New(messages.MsgErrorJSONOutputFilePathNotValid + err.Error())
+		return fmt.Errorf("%s %e", messages.MsgErrorJSONOutputFilePathNotValidUnknown, err)
```
Comment (matheusalcantarazup) [inline internal/usecases/cli/cli.go:131]: This format %e is correct here? According to [fmt](https://pkg.go.dev/fmt) package the %e is fore scientific notation, e.g. -1.234456e+78. I think that we should use %w to [wrap](https://pkg.go.dev/errors) the error

**Review by matheusalcantarazup — CHANGES_REQUESTED** [review]

File: internal/usecases/cli/cli.go (hunk 1 above)
Comment (wiliansilvazup) [reply internal/usecases/cli/cli.go]: done

**Review by nathanmartinszup — APPROVED** [review]

File: internal/usecases/cli/cli.go [hunk 2]
```diff
@@ -125,10 +125,14 @@ func (au *UseCases) validateTextOutputFilePath(cfg *config.Config) error {
...
+		return fmt.Errorf("%s %s", messages.MsgErrorJSONOutputFilePathNotValidExtension, extension)
 	}
-	if output, err := filepath.Abs(cfg.JSONOutputFilePath); err != nil || output == "" {
-		return errors.New(messages.MsgErrorJSONOutputFilePathNotValid + err.Error())
+	output, err := filepath.Abs(cfg.JSONOutputFilePath)
+	if err != nil {
+		return fmt.Errorf("%s %s", messages.MsgErrorJSONOutputFilePathNotValidUnknown, err.Error())
+	}
+	if output == "" {
+		return fmt.Errorf("%s", messages.MsgErrorJSONOutputFilePathNotValidUnknown)
```
Comment (matheusalcantarazup) [inline internal/usecases/cli/cli.go:135]: Use `errors.New(messages.MsgErrorJSONOutputFilePathNotValidUnknown)` instead.

**Review by matheusalcantarazup — CHANGES_REQUESTED** [review]

File: internal/usecases/cli/cli.go (hunk 2 above)
Comment (wiliansilvazup) [reply internal/usecases/cli/cli.go:135]: is equal

Comment (wiliansilvazup) [reply internal/usecases/cli/cli.go:135]: i beleave this isn't necessary

Comment (matheusalcantarazup) [reply internal/usecases/cli/cli.go:135]: I don't think that is equal, `fmt.Errorf` is for string formatting and create an error, and `error.New` is just for creating an error.

**Pull request merged**